src/components/Map.ts

```diff
@@ -244,6 +249,24 @@ export class Map extends Component<MapProps, MapState> {
             && !(lat === 0 && lng === 0);
     }
 
+    private setCenter() {
+        if (this.props.defaultCenterLatitude && this.props.defaultCenterLongitude) {
```

Comment:
Wonder if the priority is correct, should `bounds` before center lon lat and adres?

Reply:
Priority is very correct :). No need to use bound if the user wants location to be in a certain place